include/cufinufft.h

```diff
@@ -19,13 +19,13 @@ int cufinufft_makeplan(int type, int dim, const int64_t *n_modes, int iflag, int
 int cufinufftf_makeplan(int type, int dim, const int64_t *n_modes, int iflag, int ntr, float eps,
                         cufinufftf_plan *d_plan_ptr, cufinufft_opts *opts);
 
-int cufinufft_setpts(cufinufft_plan d_plan, int M, double *h_kx, double *h_ky, double *h_kz, int N, double *h_s,
-                     double *h_t, double *h_u);
-int cufinufftf_setpts(cufinufftf_plan d_plan, int M, float *h_kx, float *h_ky, float *h_kz, int N, float *h_s,
-                      float *h_t, float *h_u);
+int cufinufft_setpts(cufinufft_plan d_plan, int M, double *d_x, double *d_y, double *d_z, int N, double *d_s,
```

Comment:
It looks like these are "d_kx" etc in src/cuda/cufinufft.cu which is close enough.

Reply:
Better to be consistent. Fixed.